ai-service/src/ai_service/handlers/answer.py

```
@@ -0,0 +1,71 @@
+                "Here is some relevant context from the uploaded project:\n"
+                f"{context}\n\n"
+                "User question:\n"
+                f"{user_question}\n\n"
+                "Answer in detail using the project as context. If context isn't relevant, fall back to general reasoning."
+            )
+            logger.info("User question: %s", user_question)
+            logger.info("Most relevant code snippet(s): %s", context)
+
+        answer = ollama_client.chat_with_ollama(prompt)
+        logger.info("LLM answer: %s", answer)
+        return answer
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)